tests/support/mod.rs

```diff
@@ -50,6 +50,25 @@ use self::tower_service::{NewService, Service};
 pub const ENV_TEST_PATIENCE_MS: &'static str = "RUST_TEST_PATIENCE_MS";
 pub const DEFAULT_TEST_PATIENCE: Duration = Duration::from_millis(15);
 
+/// By default, disable logging in modules that are expected to error in tests.
+const DEFAULT_LOG: &'static str =
```

Comment:
This seems reasonable for now --- in the future, I think it would be ideal to capture all logging during tests and then only print it on test failures?

Reply:
:+1: sounds like a job for... tokio-trace?